libia2/test/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(permissive_mode_test permissive_mode.c)
+target_link_libraries(permissive_mode_test libia2)
```

Comment:
Test failures on ubuntu say we need libdl here too.

Is there a reason not to use our `define_test` infrastructure? It includes things like making sure our padded libc is used and propagating the build-system `LIBIA2_INSECURE` into the C preprocessor `LIBIA2_INSECURE`.

Reply:
I didn't want to actually run the rewriter or wrap anything. TBH I don't know how that macro works anymore and I didn't feel like figuring it out.